src/screens/auth/styles.ts

```diff
@@ -94,6 +91,18 @@ export const DarkGrayText = styled.Text`
   color: ${Colors.darkGray};
 `;
 
+export const BottomMargin = styled.View`
+  margin-bottom: 20px;
+`;
```

Comment:
This is not good practice for styling! Instead of having a bottom margin component, add the bottom margin/padding to the container.